indico/migrations/versions/20210215_1052_f26c201c8254_add_complete_registration_attach_ical_.py

```diff
@@ -0,0 +1,25 @@
+"""Add complete_registration_attach_ical to RegistrationForm
```

Comment:
Ah, I see you did that here nicely. Just one small thing: The trailing `_` in the filename is a bit ugly, i'd remove that.